### PR TITLE
Adding basic linting.

### PR DESCRIPTION
**Pull request by fmizzell**
Connects #13
**QA**
- Run `dktl phpcs --standard=Drupal,DrupalPractice --extensions=php,module,inc,install,test,profile,theme,info /var/www/dkan/modules/dkan/<some_module>`
- If you want to fix some of the issues, run `dktl phpcbf --standard=Drupal,DrupalPractice --extensions=php,module,inc,install,test,profile,theme,info /var/www/dkan/modules/dkan/<some_module>`
- Run `dktl test:lint dkan/modules/dkan/dkan_datastore dkan/modules/dkan/dkan_dataset`
- Run `dktl test:lint-fix dkan/modules/dkan/dkan_datastore dkan/modules/dkan/dkan_dataset`

### CODE REVIEW

**Comment by dafeder** [comment]
@fmizzell the way I imagined it when I wrote the original ticket was the options except for the path should be default, and it should default to current dir so you should be able to just type `dktl test:lint` and get something.

**Comment by fmizzell** [comment]
@dafeder The problem with the approach you are proposing is that it makes it really hard to get out of the constraints you are imposing. For example if I have a library that I wan to lint that uses PSR2, it would be very hard for us to return that power to the user.

**Comment by dafeder** [comment]
@fmizzell can't we have the arguments get default values and then you just set them differently if you want to?

**Comment by fmizzell** [comment]
The final decision is to make the commands work with some defaults, but also expose phpcs directly.

File: src/Command/TestCommands.php [hunk 1]
```diff
@@ -90,4 +91,74 @@ public function testPhpunit(array $args)
...
+        }
+        $task->run();
+    }
+
+    /**
+     * Proxy to phpcbf.
+     */
+    public function phpcbf(array $args) {
+        $dktl_dir = Util::getDktlDirectory();
+        $phpcs_command = "{$dktl_dir}/vendor/bin/phpcbf";
```
Comment (dafeder) [inline src/Command/TestCommands.php]: can we make this `$phpcbf_command`?

File: src/Command/TestCommands.php [hunk 2]
```diff
@@ -90,4 +91,74 @@ public function testPhpunit(array $args)
...
+    }
+
+    /**
+     * Preconfigured linting for paths inside of the repo.
+     */
+    public function testLint(array $paths) {
+        $dktl_dir = Util::getDktlDirectory();
+        $project_dir = Util::getProjectDirectory();
+
+        $phpcs_command = "{$dktl_dir}/vendor/bin/phpcs";
```
Comment (dafeder) [inline src/Command/TestCommands.php]: Maybe create a class property or get method for phpcs_command so we don't have this redundant code?

Comment (dafeder) [reply src/Command/TestCommands.php]: ...although actually, wouldn't it make more sense to re-use `$this->phpcs`?

**Pull request merged**